### PR TITLE
Adding RewriteOptions parameter

### PR DESCRIPTION
**Pull request by gmodules**

### CODE REVIEW

File: manifests/vhost.pp [hunk 1]
```diff
@@ -82,6 +82,7 @@
   $headers                     = undef,
   $request_headers             = undef,
   $rewrites                    = undef,
+  $rewriteoptions              = undef,
```
Comment (igalic) [inline manifests/vhost.pp]: this should follow the other options, and be
```puppet
  $rewrite_options             = undef,
```

File: templates/vhost/_rewrite.erb [hunk 2]
```diff
@@ -1,6 +1,9 @@
 <%- if @rewrites -%>
   ## Rewrite rules
   RewriteEngine On
+  <%- if @rewriteoptions %>
+  RewriteOptions <%= rewriteoptions %>
```
Comment (igalic) [inline templates/vhost/_rewrite.erb]: i'd also make this
```ruby
[@rewrite_options].flatten.join(' ')
```
this way we can pass both, a single string, or an array of strings, transparently.

**Comment by igalic** [comment]
General comments:

* please [squash](http://www.git-scm.com/book/en/Git-Tools-Rewriting-History#Squashing-Commits) this down, once you're done with it.

* We also always want tests and docs for each new feature.

* your commits have an inconsistent commit author. you may wanna consolidate those, and/or add @gmodules' email address to your github settings so it's associated with your github account.

**Comment by gmodules** [comment]
Request with the requested changes.

**Comment by mhaskel** [comment]
@gmodules we still need tests for this. Thanks!

**Comment by jonnytpuppet** [comment]
ping @gmodules is there any update on this? We'd love to see some tests written for this so we can merge this! Thanks!

**Comment by gmodules** [comment]
We'll do the tests soon. Sorry about delay

**Comment by igalic** [comment]
closing for lack of progress…

if any interest flares up again, please re-submit!

**Pull request closed without merging**